integration_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -Eemo pipefail
+# This script is meant to be run inside https://hub.docker.com/_/docker
+# See .github/workflows/ci.yml:18 for how to run this yourself
+
+# Wait for the docker daemon to start up
```

Comment:
The convention in this codebase (which is unfortunately not documented anywhere...I should create a style guide) is to punctuate comments that are sentences, and leave other comments unpunctuated. So, for example, this comment should be punctuated (`# Wait for the docker daemon to start up.`), but some of the ones below (`# ~5.5MB`) would not.

I think I'd also capitalize `Docker` here since it's the product name, rather than the command name.